apps/cookbook/src/app/examples/calendar-example/calendar-locales-examples.ts

```diff
@@ -0,0 +1,66 @@
+export const moduleProviderExample = `import {registerLocaleData } from '@angular/common';
+import localeData from '@angular/common/locales/en-DK';
+import { LOCALE_ID, NgModule } from '@angular/core';
+
+registerLocaleData(localeData);
+
+@NgModule({
+    ...
+    providers: [
+        { provide: LOCALE_ID, useValue: 'en-DK' },
+    ],
+    ...
+})
+export class AppModule {}
+`;
+
+export const componentProviderExample = `import { Component, LOCALE_ID } from '@angular/core';
+import { Locale } from '@kirbydesign/designsystem/components/calendar/calendar.component';
```

Comment:
shouldn't the import be like: `import { Locale } from '@kirbydesign/designsystem';`

Reply:
`Locale` wasn't exported through the calendar barrel file, but I have added it now and changed the import in the examples :)